chroot-part.sh

```diff
@@ -43,7 +43,8 @@ function kernel_config_set()
 }
 
 echo --- Configuring Portage
-
+# ldconfig resolves error "grep: error while loading shared libraries: libpcre.so.1: cannot open shared object file: No such file or directory" after chrooting from other linux than life CD
+ldconfig 
```

Comment:
Nevajadzētu neko nočakarēt arī ja nevajag, pieliku. https://github.com/kristapsk/gentoo-install/commit/2f85f495542a2e4addd2c34cc0b391846c62526c

Reply:
Laikam pirksts nodrebēja, nebija domāts.

Reply:
Tad šo tomēr nevajag? Revertot?

Reply:
Nepievieno

Reply:
Nu es jau pievienoju / iekommitoju, links augstāk, varu revertot, ja tas tomēr nebija vajadzīgs.

Reply:
No LiveCD šito toč nevajag, nekad nav bijusi tāda problēma.